lib/openstates/api.rb

```diff
@@ -15,6 +15,11 @@ def bills(options = {})
         end
       end
 
+      if options[:id]
+        url << "#{options[:id]}/"
+        options.delete(:id)
+      end
```

Comment:
Maybe we name this "official bill id" `official_bill_id` instead of just id. The `bills` method can check for this option and create the url using it and fall back on the previous state/session/bill_id method of looking it up. Sound reasonable?